.github/workflows/docker.yml

```diff
@@ -1,26 +1,42 @@
 name: Docker
 
 on:
-  pull_request:
+  push:
+    branches-ignore:
+      - 'master'
+      - 'release**'
+      - 'backport**'
     paths:
-    - 'Dockerfile'
-    - '.github/workflows/docker.yml'
+      - "**"
```

Comment:
I think `**` as path is implicit anyway, it should not be necessary.

Reply:
Do you think it makes sense to ignore all the directories that start with a "."? otherwise it will also build branches if we change the pipelines or change something that might be external to the code